Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteScrollingTreeMac.mm

```diff
@@ -544,13 +544,13 @@ static bool layerEventRegionContainsPoint(CALayer *layer, CGPoint localPoint)
 #if ENABLE(WHEEL_EVENT_REGIONS)
 OptionSet<EventListenerRegionType> RemoteScrollingTreeMac::eventListenerRegionTypesForPoint(FloatPoint point) const
 {
-    auto* rootScrollingNode = rootNode();
+    RefPtr rootScrollingNode = dynamicDowncast<ScrollingTreeFrameScrollingNodeMac>(rootNode());
```

Comment:
Shouldn't this be a plain downcast? In that rootNode() is either null or a ScrollingTreeFrameScrollingNodeMac per the prior code?

Reply:
Yes, I agree. I think we should use downcast to match old behavior.